bpf/lib/nodeport.h

```diff
@@ -1001,7 +1001,7 @@ static __always_inline int rev_nodeport_lb6(struct __ctx_buff *ctx, __u32 *ifind
 #endif
 
 	ret = lb6_extract_tuple(ctx, ip6, &l4_off, &tuple);
-	if (ret) {
+	if (ret < 0) {
```

Comment:
```suggestion
        if (IS_ERR(ret)) {
```

Reply:
I agree we should be using this style in general, but there are several other places in this file that are not yet using this. @julianwiedmann , would you mind following up on this separately to update these and other remaining `if (ret < 0)` cases in this file?

Reply:
@joestringer yeah I plan to spend a few cycles on the wider error handling scheme.

Think with a bit of effort we should be able to banish `CTX_ACT_DROP` from all/most our own code (just return a proper negative `DROP_*` everywhere), and translate all helper calls (eg. for `ctx_redirect()`, don't we want some `DROP_MISSED_BPF_REDIRECT` visibility anyway?). At which point `IS_ERR()` might just turn into a  simple `ret < 0` check.